commit 6ea42ebb24663f22da2dd6c4129933427a9e16a5
Author: David Scott <dave.scott@eu.citrix.com>
Date:   Thu Apr 23 20:55:01 2015 +0100

    Add an explicit version constraint on message-switch
    
    This should cause a build to fail earlier avoiding a strange
    type error.
    
    Signed-off-by: David Scott <dave.scott@eu.citrix.com>

diff --git a/_oasis b/_oasis
index a0d5b2c..7db7067 100644
--- a/_oasis
+++ b/_oasis
@@ -13,7 +13,7 @@ Library xcp
   Findlibname:        xcp
   Modules:            Cohttp_posix_io, Posix_channel, Open_uri, Xcp_client, Xcp_service, Xcp_channel, Xcp_channel_protocol, Syslog, Debug
   CSources:           syslog_stubs.c
-  BuildDepends:       cmdliner, uri, re, cohttp, xmlm, unix, sexplib, sexplib.syntax, rpclib, rpclib.syntax, rpclib.xml, threads, message_switch, message_switch.unix, fd-send-recv, xcp-inventory, xapi-backtrace
+  BuildDepends:       cmdliner, uri, re, cohttp, xmlm, unix, sexplib, sexplib.syntax, rpclib, rpclib.syntax, rpclib.xml, threads, message_switch (>= 0.11.0), message_switch.unix, fd-send-recv, xcp-inventory, xapi-backtrace
 
 Library xcp_storage
   CompiledObject:     best
diff --git a/setup.ml b/setup.ml
index 8182c5e..09c1b68 100644
--- a/setup.ml
+++ b/setup.ml
@@ -1,7 +1,7 @@
 (* setup.ml generated for the first time by OASIS v0.3.1 *)
 
 (* OASIS_START *)
-(* DO NOT EDIT (digest: 12f9e81c549a58a2d7c373b5a9257a05) *)
+(* DO NOT EDIT (digest: 7c78c15ac41edb6453d49c81ae9a122f) *)
 (*
    Regenerated by OASIS v0.4.5
    Visit http://oasis.forge.ocamlcore.org for more information and
@@ -6930,7 +6930,9 @@ let setup_t =
                            FindlibPackage ("rpclib.syntax", None);
                            FindlibPackage ("rpclib.xml", None);
                            FindlibPackage ("threads", None);
-                           FindlibPackage ("message_switch", None);
+                           FindlibPackage
+                             ("message_switch",
+                               Some (OASISVersion.VGreaterEqual "0.11.0"));
                            FindlibPackage ("message_switch.unix", None);
                            FindlibPackage ("fd-send-recv", None);
                            FindlibPackage ("xcp-inventory", None);
@@ -7309,7 +7311,7 @@ let setup_t =
        };
      oasis_fn = Some "_oasis";
      oasis_version = "0.4.5";
-     oasis_digest = Some "\021\128\223Uc\005aZ\182%\214v\229\159'\203";
+     oasis_digest = Some "\230\173V\014Y\252\200\2235\006j \224\018\156\231";
      oasis_exec = None;
      oasis_setup_args = [];
      setup_update = false
@@ -7317,6 +7319,6 @@ let setup_t =
 
 let setup () = BaseSetup.setup setup_t;;
 
-# 7321 "setup.ml"
+# 7323 "setup.ml"
 (* OASIS_STOP *)
 let () = setup ();;
